deepinv/physics/tomography.py

```diff
@@ -185,3 +201,355 @@ def A_adjoint(self, y, **kwargs):
             if self.normalize:
                 output = output / output.shape[-1]
             return output
+
+
+class TomographyWithAstra(LinearPhysics):
+    r"""Computed Tomography operator with `astra-toolbox <https://astra-toolbox.com/>`_ backend.
+    It is more memory efficient than the :class:`deepinv.physics.Tomography` operator and support 3d geometries.
+    See documentation of :class:`deepinv.physics.functional.XrayTransform` for more
+    information on the ``astra`` wrapper.
+
+    Mathematically, it is described as a ray transform
+    :math:`R` which linearly integrates an object :math:`x` along straight
+    lines
+
+    .. math::
+        y = Rx
```

Comment:
Is R standard? Just because our deepinv convention is R=\inverse{} ie the reconstruction operator, and A=\forw{} ie the forward operator